dcm4che-imageio-test/src/test/java/org/dcm4che3/imageio/dcm/TestDicomImageReader.java

```diff
@@ -136,7 +137,19 @@ public void testReadUncompressedPostPixelData_fromImageInputStream() throws IOEx
             testReadPostPixelData(is);
         }
     }
-    
+
+    /**
+     * This test previously throw an out of memory exception, shows that the changes work.
+     * @throws IOException
+     */
+    @Test
+    public void testReadUncompressedingleFrame_fromImageInputStream() throws IOException {
```

Comment:
Typo: "testReadUncompressedingleFrame_fromImageInputStream" -> "testReadUncompressedSingleFrame_fromImageInputStream"

Reply:
I have corrected the name of the test.